src/test/java/com/marizueva/laboratory/hw1/calculatort_tests/CheckValueIsNegativeOrPositive.java

```diff
@@ -0,0 +1,37 @@
+package com.marizueva.laboratory.hw1.calculatort_tests;
+
+import com.epam.tat.module4.Calculator;
+import com.marizueva.laboratory.hw1.calculatort_tests.base_test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CheckValueIsNegativeOrPositive extends BaseTest {
+
+    @Test
+    public void checkIsNegativeFalse() {
+        boolean negative = calculator.isNegative(-0);
+
+        Assert.assertFalse(negative);
```

Comment:
If you start using assertions from the `Hamcrest` please use them in all code. It is not good idea mix them with the `testing` standard asserts

Reply:
corrected for this class